.env-issuer.sample

```diff
@@ -12,22 +13,47 @@ ISSUER_LOG_LEVEL=-4
 ISSUER_LOG_MODE=2
 ISSUER_API_AUTH_USER=user-issuer
 ISSUER_API_AUTH_PASSWORD=password-issuer
-ISSUER_KEY_STORE_ADDRESS=http://vault:8200
-ISSUER_KEY_STORE_PLUGIN_IDEN3_MOUNT_PATH=iden3
-# Could be either localstorage or vault
-ISSUER_KMS_PLUGIN=localstorage
+
+# --------------------------------------------------------------------------------
+# KMS configuration
+# --------------------------------------------------------------------------------
+# Could be either [localstorage | vault] (BJJ) and [localstorage | vault | aws ] (ETH)
+ISSUER_KMS_BJJ_PLUGIN=vault
```

Comment:
In my opinion, the name `ISSUER_KMS_BJJ_PROVIDER` is clearer. A plugin is something that is embedded in existing code (like custom plugins for vault) otherwise, the provider is a separate service (vault, was kms) that provides some service.  I think a provider is a better variant.